source/Projectile.js

```diff
@@ -15,8 +15,26 @@ export default class Projectile extends Pixi.Sprite {
       this.projectileType = projectileType
       this.friendly = (friendly === undefined) ? true : friendly
       if (this.friendly) {
-          Projectile.FriendlyInventory.push(this)
-          this.texture = PIXI.loader.resources.projectile.texture
+          if(this.projectileType == "bullet") {
+              Projectile.FriendlyInventory.push(this)
+              this.texture = PIXI.loader.resources.projectile.texture
+          } else if(this.projectileType == "laser") {
+              Projectile.FriendlyInventory.push(this)
+              this.texture = PIXI.loader.resources.laser.texture
+              console.log("Laser")
+          } else if(this.projectileType == "piercinglaser") {
+              this.piercing = true
+              Projectile.FriendlyInventory.push(this)
+              this.texture = PIXI.loader.resources.plaser.texture
+              console.log("piercingLaser")
+          } else if(this.projectileType == "superlaser") {
+              this.piercing = true
+              this.despawn = true
+              this.spawnTime = Date.now()
+              Projectile.FriendlyInventory.push(this)
+              this.texture = PIXI.loader.resources.slaser.texture
+              console.log("superLaser")
+          }
       } else {
```

Comment:
Can we pull up the common behavior from all these else-if's? Particularly adding to the FriendlyInventory array.  And remove all the `console.log`'s (unless you want to place the logging behind some control to be able to turn it on and off without having to individually comment/remove the statements). Also, little neat thing is that object properties can be accessed using a dictionary syntax, like `PIXI.loader.resources.laser.texture` could be accessed using `PIXI.loader.resources["laser"].texture`, which means you could actually make choosing the textures dynamic instead of being placed inside the else-if branching. Not really necessary, just good to know for the future.